install.bat

```diff
@@ -0,0 +1,77 @@
+@ECHO off
+
+ECHO "Welcome to the setup for mentorship-backend repo."
+
+ECHO "This installation requires Python version above 3.0. Make sure you have it installed."
+
+SET /P check_repo="Do you want to clone the repository [y/n] : "
+IF %check_repo%=="y" ( ECHO "Cloning the repository"
+git clone https://github.com/systers/mentorship-backend.git 
+) ELSE ( 
+ECHO "Make sure that you have the repository and the install.bat file in the same folder!" 
+)
+
+cd mentorship-backend
+
+ECHO "Installing the virtual environment - 'venv'"
+pip3 install virtualenv
+virtualenv venv
```

Comment:
Give it a python version (python3) which it will use for creating virtual environment.
Problem arises when python2 is used by default, so explicitly mention the usage of python3.

Reply:
I have made this correct!